app/src/main/java/nerd/tuxmobil/fahrplan/congress/designsystem/colors/ColorSchemes.kt

```diff
@@ -16,6 +16,7 @@ internal fun darkColorScheme() = androidx.compose.material3.darkColorScheme(
     onSurface = colorResource(R.color.text_primary), // used by ListItem -> headlineContent
     onSurfaceVariant = colorResource(R.color.text_secondary), // used by SearchBarDefaults.InputField placeholder, ListItem -> overlineContent
     inverseOnSurface = colorResource(R.color.session_item_text_on_highlight_background), // used by SessionCard
+    outlineVariant = colorResource(R.color.colorAccent), // used by SearchHistoryList -> ButtonOutlined -> border
```

Comment:
TODO: Change based on this comment from @cketti https://github.com/EventFahrplan/EventFahrplan/pull/791#discussion_r2505900089.